src/lib/components/chat/SettingsModal.svelte

```diff
@@ -732,8 +734,24 @@
 									class="w-full h-2 rounded-lg appearance-none cursor-pointer dark:bg-gray-700"
 								/>
 							</div>
+							<div>
+								<label for="steps-range" class=" mb-2 text-sm font-medium flex justify-between">
+									<div>Context Length</div>
+									<div>
+										{num_ctx}
+									</div></label
+								>
+								<input
+									id="steps-range"
+									type="range"
+									min="0"
```

Comment:
Are there better values for min/max? Min of 0 doesn't make all that much sense to me.

Reply:
I guess it just depends on how much previous context someone wants during a chat

Reply:
The min value for OpenAI Playground value seems to be 1. Once you update the min value to 1, I'll try to run it on couple of my machines then merge to main, Thanks!

Reply:
Updated